src/components/ProjectsDropdown/index.tsx

```diff
@@ -7,7 +7,7 @@ import { useLocalization } from 'src/providers';
 type ProjectsDropdownProps<T extends { projectId?: number } | undefined> = {
   allowUnselect?: boolean;
   availableProjects: Project[] | undefined;
-  label?: string | undefined;
+  label?: string | false | undefined;
```

Comment:
This is fine.. would a `label=''` in the client render the same result?

Reply:
Unfortunately it doesn't work because it is falsey, so it goes with the default `strings.PROJECT`. I will remove the `false` case though and just check for empty string.